src/client/core/evaluator.ts

```diff
@@ -32,18 +32,18 @@ const factories = {
 }
 export const math = create(factories, config)
 
-export function operation(
+export function evaluateOperation(
```

Comment:
Consider the more concise...
```suggestion
export function evaluate(
```